client/src/components/formHelpers/index.js

```diff
@@ -20,7 +20,7 @@ export function formatUrlValues(values, options) {
     let value = values[key];
     const nullOrWarning = composeValidators(
       localhostValidator,
-      editorValidator
+      key === 'githubLink' ? null : editorValidator
```

Comment:
Just a thought, here - should we add validation on the GitHub submission box to ensure that it is a GitHub URL and not another live link?

Reply:
That is a bit beyond the scope of anything we can reasonably control. Reason being: I specifically set this up so Campers can submit a `repl.it/@ShaunSHamilton/...` link to the GitHub Solution, because we do say you can submit '_any code link in the GitHub input_' (highly paraphrased).

So, I do not know what regex would test for _any code that is not to a live version of the code_. Unless, you would want a validation to ensure Campers do not submit the same link in both?

Reply:
That could be a good approach - also possibly reversing the regex assertion.

Either way, I'm mostly thinking out loud here and this definitely isn't required for this PR.